### ISSUE TITLE
As a reader, I want to see an overview of an edition

### ISSUE THREAD

**Issue by jamiefolsom** [SHOREdevelopers/COVE#49]
#### Assumptions
- Editions may be public, or private, so this index page should deal correctly with authenticated and non-authenticated users.
- Each editions is owned by an admin or editorial user, and can be managed by only that user, or class of users.
- Editions can contain documents, and other resources (timelines, maps, images, etc), and provide summaries of their contents via this index page.

**Comment by daverett** [comment]
Do you mean something like a table of contents or navigation menu?

**Comment by jamiefolsom** [comment]
Right. Like this page: http://dev-rc-distro.pantheonsite.io/edition/sonnet -- if you consider this feature complete, let's move it to delivered, with a cover letter, so Steve can test. If not, we can use this ticket to flesh out what's left to be done. Sound good?
